trust-sdk/trust-sdk-specs.md

```diff
@@ -0,0 +1,119 @@
+## URL Specifications
+
+TrustSDK works by calling deeplinks between apps, this document describes URL formats for each command. Due to the SDK high customization provided by TrustSDK, clients and wallets can adapt the URLs to their own needs. 
+
+All commands follow the URL format below:
+
+```
+[scheme]://[command]?[params]&[app]&[id]&[callback]
+```
+
+Where: 
+* `[scheme]` is the wallet's deeplink scheme. Default to `trust`
+* `[command]` is the command to be executed
+* `[params]` is the command parameters encoded as url query items.
+* `[id]` is the unique command id. The value is later used to resolve the callback for the command. Can be any incrementing integer.
+* `[app]` callback URL scheme. 
+* `[callback]`callback path. Default to `sgn_sign_result`
+
+### Sign
+
+Sign command is used to sign and broadcast transactions following [wallet-core](/wallet-core/wallet-core.md) protobuf models encoded in base64.
+
+#### Command
+* Command: `sdk_sign`
+* Parameters:
+  * `coin`: BIP44 coin code
+  * `data`: wallet-core protobuf model binary encoded in base64
+  * `send`: flag that indicates if the wallet should broadcast the transaction. Possible values: `true`, `false`.
+  * `meta`: transaction metadata. This attribute is optional and encoded as a dictionary. Check how it's encoded in [Dictionary Encoding](#dictionary-encoding).
+    * `meta.__name`: transaction metadata type. Possible values: `dApp`
```

Comment:
dApp or dapp?
```suggestion
    * `meta.__name`: transaction metadata type. Possible values: `dapp`?
```

Reply:
`dApp`

Reply:
the example request is still `dapp`

Reply:
case-insensitive check is a good idea in such cases (just to consider)